.github/CONTRIBUTING.MD

```diff
@@ -1,60 +1,61 @@
+6. Do not directly interact with the Minecraft protocol.
+   - This includes the usage of ProtocolLib or Netty.
+
+# About dependencies
+For the sanity of the rest of us, and the maintainability of the project, please adhere to the following guidelines when attempting to include a dependency or an third party library.
+
+1. Libraries need to be approved by the maintainer before being merged. We recommend you ask before commiting.     
+2. Prevent `ClassNotFound` exceptions.
    - Do not reference a plugin in a loaded class, unless that plugin is enabled.
-2. Do not embed your own plugins within Towny
-   - Dependencies should be within the 'Provided' scope.
-   - They will also need peer-review.
-3. No "Black Box" or Obufscated dependencies.
-   - All dependencies should have their source open and auditable.
-
-## Updating Language Files
-> As of August 2021, translation for the Towny project is being done on [Crowdin](https://crowdin.com/project/townyadvanced).
-
-The language files are stored in the resources section of the project. (`resources/lang/"language".yml`).
-If you want to create a new language file, contact us on [Discord](https://discord.gg/gnpVs5m).
-
-If you find that a line has multiple variables which would be in a different order in your language, you can change them to `%Variable number$s` to change their order. 
-An example of this: `%4$s` will **always use the 4th variable**, no matter where it is placed on the line.
-
-You can read more about this on [Java Platform SE's Javadocs regarding the Formatter](https://docs.oracle.com/javase/7/docs/api/java/util/Formatter.html).
-
-<!--Pre-Crowdin language file tutorial-->
-<!--To update / change an existing language file, fork this repo and then edit the "language".yml in your fork.    
-Pull-Request your updated version. Please try to keep the title simple and include the version number:    
-e.g "German Translation v0.42"    
-If you are creating a new language file, feel-free to use either the english translation e.g "norwegian.yml" or the global placeholder e.g "es-mx"    
-If you are editing a language which contains the '(apostrophe) character such as French, please make sure you run it all through a YAML Parser,    
-You have to double up the '' when you want one ' to appear. It may help to do this anyways, as other characters may also require an escape sequence.-->
+3. Do not add plugin support directly into Towny **unless** it can benefit the project (such as economy/permission bridges).
+   - For Towny support, you should be using the [`TownyAPI`](https://github.com/TownyAdvanced/Towny/wiki/TownyAPI) instead.
+   - Even if you believe it can benefit the project, please ask the maintainer first.
+4. Do not embed dependencies directly into Towny.
+   - All dependencies should use the `provided` scope instead. This means that the dependency will be provided by the plugin JAR (e.g. Vault), or the server JAR (e.g. Adventure).
```

Comment:
```suggestion
   - All dependencies should use the `provided` scope instead. This means that the dependency will be provided by the plugin JAR (e.g. Vault), or the server JAR (e.g. Essentials, Citizens).
```

Reply:
Not sure what you mean here, by server JAR I'm talking about libraries included in paper, spigot, yk, adventure and gson for example